### PR TITLE
Adds a ghost role to the gas station ruin on Lavaland

### PR DESCRIPTION
**Pull request by EEASAS**
## About The Pull Request
Adds a new ghostrole to the Lizard's Gas Station ruin on Lavaland.
They have enough supplies to expand, like a pickaxe, coat, oxygen tank, and a pipe rifle with 3 rounds.
They can also stay there for the entire round due to the abundance of food and power.
gimmick idea: try to find lavaland syndicate and deliver food to them for extra supplies
## Why It's Good For The Game
Ghost roles are good for well, ghosts to "soft-respawn" and this one gives an opportunity to roleplay
instead of just hunting megafauna left and right
## Changelog
:cl:
add: Added a new ghost role to lavaland, specifically the gas station ruin
/:cl:

### CODE REVIEW

File: code/modules/mob_spawn/ghost_roles/mining_roles.dm [hunk 1]
```diff
@@ -339,3 +339,25 @@
...
+///Lavaland Gas Station Worker
+/obj/effect/mob_spawn/ghost_role/human/lavaland_gasstation
+	name = "Lavaland Gas Station Worker"
+	desc = "Seems like there's a lizardperson inside, peacefully sleeping."
+	mob_species = /datum/species/lizard
+	icon = 'icons/obj/machines/sleeper.dmi'
+	icon_state = "sleeper"
+	prompt_name = "a gas station worker"
+	you_are_text = "You are a worker at a Lizard's Gas Station close to a mining facility."
+	flavour_text = "Your employer, however, failed to realize that there are hostile megafauna and tribes in the area, so make sure that you can defend yourself. Also sell stuff to people, ocasionally."
```
Comment (Sadboysuss) [inline code/modules/mob_spawn/ghost_roles/mining_roles.dm:352]: Ocasionally -> occasionally

**Comment by DaCoolBoss** [comment]
I feel like ghost roles should have more of a gameplay loop than this. They need something to do if they're isolated so they don't get bored AFKing, and they need a way for outsiders to involve themselves in the ghost role's gameplay. Retail is a fine gimmick but this role being disconnected from the economy systems mean there's no gameplay incentive or depth to this.

My suggestion to add depth is this. Add a variant of the chef's robot portal to the ruin. Make the robots request various junk foods that you want stocked in the gas station, and reduce their patience values by a lot. Give the worker a console or remote to order replacement items to sell, items are drop podded in the same way express orders work for kitchen. Make profit margins small, since we want to encourage trading with crew.

**Comment by EEASAS** [comment]
> gameplay loop snip

sorry i dont know how to code

**Comment by DaCoolBoss** [comment]
Okay if this gets merged I'll work on it myself.

**Comment by ZephyrTFA** [comment]
Please raise your GNP

**Pull request closed without merging**